homeassistant/components/matter/sensor.py

```diff
@@ -92,7 +90,6 @@ def _update_from_device(self) -> None:
         platform=Platform.SENSOR,
         entity_description=MatterSensorEntityDescription(
             key="FlowSensor",
-            name="Flow",
```

Comment:
This entity description seems to be wrong on multiple levels.

- The `water` device class is used for volume, not for flow rate.
- Removing "Flow" here, makes it being named "Water" instead of a flow rate sensor.

Reply:
oh that is not good, it should be a flow rate sensor. Shall I put back the flow name here ? Or do we have another device class ?